docs/contracts/index.md

```diff
@@ -26,6 +26,11 @@ if __name__ == '__main__':
     divide(10, 0)  # Preconditions on y violated
 ```
 
+```{note}
```

Comment:
You can remove this note, I don't want to distract from the main flow of the page here. It's good to show an example, but this should be done in the CLI section below.